androiddemo/src/test/kotlin/com/example/androiddemo/DepsTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.androiddemo
+
+import com.swmansion.starknet.crypto.Poseidon
+import com.swmansion.starknet.crypto.StarknetCurve
+import com.swmansion.starknet.crypto.starknetKeccak
+import com.swmansion.starknet.data.types.Felt
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.OkHttpClient
+import org.junit.Assert
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Test
+
+internal class DepsTest {
+    @Test
+    fun `okhttp client`() {
+        val client = OkHttpClient()
+        assertNotNull(client)
+    }
+
+    @Test
+    fun `json serialization`() {
+        val json = Json { ignoreUnknownKeys = true }
+        val value: Int = json.decodeFromString("451")
+        assertEquals(451, value)
+    }
```

Comment:
They tests that all the deps are available. 
Previously, they weren't, so unit tests that include any of the deps would fail.

I did not add any new unit tests, just made sure it's possible to add those should such need appear.